deploy/README.md

```diff
@@ -5,66 +5,15 @@ This page has instructions for collecting Kubernetes logs, metrics, and events;
 <!-- TOC -->
```

Comment:
can you get rid of this and `<!-- /TOC -->` below as well, it will conflict with the TOC generator plugin I have